systems/lanl-venado/system.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import pathlib
+
+from benchpark.directives import variant
+from benchpark.system import System
+
+id_to_resources = {
+    "grace-hopper": {
+        "sys_cores_per_node": 144,
+        "sys_gpus_per_node": 4,
+    },
+    "grace-grace": {
+        "sys_cores_per_node": 144,
+    },
+}
+
+
+class LanlVenado(System):
```

Comment:
@rfhaque Do they call them "clusters" or "partitions"?  Do you also need to set "partition" in the slurm script to correspond to which partition of the machine you are wanting to use?

@alexrlongne might know.

Reply:
@pearce8 While technically these are hardware partitions, venado slurm defines its own "partitions" (standard, debug, gpu etc) corresponding to some set of resources/qos